packages/trip-details/src/fare-table.tsx

```diff
@@ -113,7 +119,16 @@ const FareTypeTable = ({
               >
                 {boldText(useGetHeaderString(col.columnHeaderKey))}
                 <br />
-                {renderFare(fare?.currency?.code, fare?.amount || 0)}
+                {fare?.amount !== undefined ? (
+                  renderFare(fare?.currency?.code, fare?.amount)
+                ) : (
+                  <>
+                    -
+                    <InvisibleA11yLabel>
+                      <FormattedMessage id="otpUi.TripDetails.legMissingFareInfo" />
```

Comment:
I think you swapped the leg and total fare by mistake...

Reply:
Oops... good catch